phylogenetic/rules/prepare_sequences.smk

```diff
@@ -3,8 +3,8 @@ This part of the workflow prepares sequences for constructing the phylogenetic t
 
 REQUIRED INPUTS:
 
-    metadata    = data/metadata.tsv
-    sequences   = data/sequences.fasta
+    metadata    = results/metadata.tsv
+    sequences   = results/sequences.fasta
```

Comment:
These were changed from `data/metadata.tsv` to `input_metadata` then to `results/metadata.tsv`. Would it be better to keep as `data/metadata.tsv` since [most workflows](https://github.com/search?q=org%3Anextstrain+path%3Aphylogenetic%2Frules%2Fprepare_sequences.smk+"data%2Fmetadata.tsv"&type=code) are already using that?

(same goes for sequences)